Translation/DatabaseFreshResource.php

```diff
@@ -2,14 +2,14 @@
 
 namespace Lexik\Bundle\TranslationBundle\Translation;
 
-use Symfony\Component\Config\Resource\ResourceInterface;
+use Symfony\Component\Config\Resource\SelfCheckingResourceInterface;
```

Comment:
then it would need something like they did in assetic bundle
https://github.com/symfony/assetic-bundle/pull/395/files